cmd/kube-controller-manager/app/controller_descriptor.go

```diff
@@ -65,8 +65,8 @@ func (r *ControllerDescriptor) GetControllerConstructor() ControllerConstructor
 	return r.constructor
 }
 
-func (r *ControllerDescriptor) GetRequiredFeatureGates() []featuregate.Feature {
-	return append([]featuregate.Feature(nil), r.requiredFeatureGates...)
+func (r *ControllerDescriptor) GetRequiredFeatureGates() []string {
+	return append([]string(nil), r.requiredFeatureGates...)
```

Comment:
It would be better to sort the gates before returning here. This will render in the help/docs and it is better to ensure that we always have the same order.